### DISCUSSION TITLE
Implementation on Tree Based Model

### DISCUSSION THREAD

**Discussion by SeptivianaSavitri** [HewlettPackard/swarm-learning#62] [Q&A]
Dear HPE team, I'd like to ask, is it possible to implement the swarm learning under the tree based model, such as xgboost, random forest, etc? thankyou

**Comment by RadhakrishnaJ** [comment]
Hello, existing Eval Swarm product does not support Nonparametric machine learning algorithms. It is in our road map for future updates.
